plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -471,7 +472,8 @@ func ClusterRoles() []rbacv1.ClusterRole {
 	}
 
 	kubeSchedulerRules := []rbacv1.PolicyRule{
-		eventsRule(),
+		// TODO: use eventsRule() once everything is migrated to v1beta1 Events API
```

Comment:
Instead, can you create "newEventsRuls()" (or sth like that, better name welcome) and use it here?